CONTRIBUTING.md

```diff
@@ -37,6 +37,47 @@ Note that we welcome all contributions, e.g., further protocol models. Just
 send us a pull request.
 
 
+Making new releases
+-------------------
+
+The Tamarin Prover is distributed through a number of channels: source and binaries through Github, and packages through Homebrew, Arch and NixOS. When a new release is cut, here's how to update the various distribution channels.
```

Comment:
is it worth commenting on when a new release should happen? (i.e. what are the criteria for when to do a release?)

Reply:
Yes, excellent point. I wrote a first draft. If you dislike its English, please feel free to fix.